firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -155,6 +167,7 @@ uint8_t moveDiagonal(int startCol, int startRow, int endCol, int endRow)
   int unitSpacesX, unitSpacesY;
   int dirX, dirY;
   int numStepsX, numStepsY;
+  int numEighthStepsX, numEighthStepsY;
   int i;
 
   // Abs ensures that numStepsX and numStepsY will be positive
```

Comment:
Can you comment that, in the following blocks, you're setting the scale and assigning the number of eighth steps for X and Y?

Reply:
replace "is set as well" with "are set"

Reply:
*numEighthSteps